crates/typst-library/src/layout/transform.rs

```diff
@@ -115,44 +111,43 @@ impl Show for Packed<RotateElem> {
     }
 }
 
-/// Scales content without affecting layout.
+/// レイアウトに影響を与えないコンテンツの拡大縮小。
 ///
-/// Lets you mirror content by specifying a negative scale on a single axis.
+/// 単一の軸で負のスケールを指定することで、コンテンツを反転表示できます。
 ///
-/// # Example
+/// # 例
 /// ```example
 /// #set align(center)
 /// #scale(x: -100%)[This is mirrored.]
 /// #scale(x: -100%, reflow: true)[This is mirrored.]
 /// ```
 #[elem(Show)]
 pub struct ScaleElem {
-    /// The scaling factor for both axes, as a positional argument. This is just
-    /// an optional shorthand notation for setting `x` and `y` to the same
-    /// value.
+    /// 位置引数として両方の軸の拡大縮小率を設定します。
+    /// これは`x`と`y`を同じ値で設定する単なる省略記法です。
```

Comment:
optionalを除いたようにjustも除いて訳してしまった方がスッキリ読めると思います。
逆に、それらのニュアンスを強く出したい場合には「これは`x`と`y`に同じ値を設定するための省略記法にすぎません。」みたいな言い方もあると思います。

```suggestion
    /// これは`x`と`y`を同じ値で設定する省略記法です。
```